src/server/qgsserversettings.cpp

```diff
@@ -333,6 +333,29 @@ void QgsServerSettings::initSettings()
                                     QVariant()
                                   };
   mSettings[ sServiceUrl.envVar ] = sWmtsServiceUrl;
+
+  // the default config cache check interval
+  const Setting sConfigCacheCheckInterval = { QgsServerSettingsEnv::QGIS_SERVER_PROJECT_CACHE_CHECK_INTERVAL,
+                                              QgsServerSettingsEnv::DEFAULT_VALUE,
+                                              QStringLiteral( "The default project cache check interval" ),
+                                              QStringLiteral( "/qgis/server_project_cache_check_interval" ),
+                                              QVariant::Int,
+                                              QVariant( 0 ),
+                                              QVariant()
+                                            };
+  mSettings[ sServiceUrl.envVar ] = sConfigCacheCheckInterval;
+
+  // the default config cache check interval
+  const Setting sDisableConfigCache = { QgsServerSettingsEnv::QGIS_SERVER_PROJECT_CACHE_STRATEGY,
+                                        QgsServerSettingsEnv::DEFAULT_VALUE,
+                                        QStringLiteral( "Project's cache strategy" ),
```

Comment:
maybe useful to list the possible values in the description

Reply:
Done.